src/app.js

```diff
@@ -78,15 +79,23 @@ app.post('/help', (req, res) =>{
         }
       );
 
-      email_sender(email)
+    // Send an email with the below
+    const options = {
+        from: 'from@from.com',
+        to: 'to@to.com',
+        subject: "You're both free now."
+    };
+
+    sendEmail(options);
```

Comment:
You'll want to wrap this in a try/catch to handle any errors that this function might throw.

```js
try {
  sendEmail(options);
} catch (error) {
  // handle error logic here 
}